### ISSUE TITLE
[ACT] Map existing rules to ACT Proposed rule afw4f7  "Text has minimum contrast"

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1150]
Map the following two existing rules to the ACT proposed rule **_afw4f7_**  [Text has minimum contrast](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/afw4f7/proposed/) 

- [x] Map [IBMA_Color_Contrast_WCAG2AA](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/IBMA_Color_Contrast_WCAG2AA.ts) Fail_1 to ACT afw4f7 (Fail Example 1, 4, 5, 8)
- [x] Map [IBMA_Color_Contrast_WCAG2AA_PV](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/IBMA_Color_Contrast_WCAG2AA_PV.ts) Potential_1 to  ACT afw4f7 (Fail Example 2, 3, 7)

See **_Details by test case_** tab in the spreadsheet [ACT vs Engine inconsistencies.xlsx](https://ibm.ent.box.com/file/1040535127435?s=9o6m83gzzc9sp8pu8lwtyesh0b8wwxln) on box.

A separate bug #1151 is opened to add a rule or extend fail case to cover low contrast content in Shadow Dom. See ACT  afw4f7 [Fail Example 6](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/afw4f7/b1a65bd18381a1ea4ad3077fd98c50368947012c.html)

**Comment by philljenkins** [comment]
This should be a simple code edit to line 45 to make the mapping to the ACT rule **_afw4f7_**
`act: [],`
